doc/HOWTO_oracle.md

```diff
@@ -13,11 +13,14 @@ Follow these instructions to set up the Oracle database to run in Docker, but th
 * Start the oracle container:
 
 ```
-docker run -d -p 1521:1521 --name nucore_db sath89/oracle-12c
+docker run -d -p 1521:1521 --name nucore_db -v /u01/app/oracle:~/oracle_data sath89/oracle-12c
```

Comment:
I could probably use some feedback on this as I did the volume setup originally through Kitematic.

Reply:
Yeah so we might need to pair on this. Running this line of code I get an error, I suspect because my docker is set up differently.

Reply:
Needs to be `$HOME`, `~` doesn't work.